merlin/dataloader/loader_base.py

```diff
@@ -29,6 +29,11 @@
 except ImportError:
     cp = np
 
+try:
```

Comment:
This could use `cudf` from `merlin.core.compat` but that's a tiny nit that need not block this PR 🤷🏻

Reply:
Once https://github.com/NVIDIA-Merlin/core/pull/244 is merged we'll be able to. (There's still one thing to fix on that PR)